packages/dd-trace/src/tracer.js

```diff
@@ -17,11 +19,24 @@ const MEASURED = tags.MEASURED
 class DatadogTracer extends Tracer {
   constructor (config) {
     super(config)
-
+    this._dataStreamsProcessor = new DataStreamsProcessor(config)
     this._scope = new Scope()
     setStartupLogConfig(config)
   }
 
+  setCheckpoint (edgeTags) {
```

Comment:
Similar to the checkpoint, this should not be on the tracer.

Reply:
Talked about it on zoom, we will keep that for now in the tracer, but not expose it to avoid too many changes here.